### ISSUE TITLE
Unable to inject library into process without libc

### ISSUE THREAD

**Issue by hammad2224** [hacksysteam/android-injector#2]
i am using emulator and tried to inject but this is the error i get
Frida version frida-core-devkit-15.1.17-android-arm.tar.xz

[+] Patching SeLinux policy
[+] Injecting library: /data/local/tmp/libagent.so in pid: 5897
[-] Error while injecting: Unable to inject library into process without libc
[+] Injection completed

**Comment by hacksysteam** [comment]
Hi @hammad2224 seems the issue is with Frida. Did you try to update Frida to latest and build this?

Currently this tool is using Frida 12.11.8 which is old now. Can you just change the frida version in the Makefile and build it again.

Thanks

**Comment by hammad2224** [comment]
Hi @hacksysteam 
I tried every version of Frida on Google people say it works with older versions but it does not.
I tried the most latest and the Google suggested older versions of Frida

**Comment by hacksysteam** [comment]
Hi @hammad2224 is it possible for you to share more details in which process you are trying to inject? What about injecting in same process on some other phone?

**Comment by hacksysteam** [comment]
@hammad2224 this turns out to be a known issue with Frida https://github.com/frida/frida/issues/936

Can you try these steps and see if it fixes the issue.

Thanks

**Comment by hammad2224** [comment]
@hacksysteam  yes i also found the same page in frida issues and someone mentioned to try 12.4.8
so i tried that version and versions mentioned on google and other posts.

i tried it on 32bit mobile then 64bit mobile then emulators it does not work 
just to let you know i carefully tried to make sure ABI was matched matched for libfrida-core.a

**Comment by ibrasec** [comment]
same here, i was able to use frida-15 on a 32bit emulator, but couldn't use it on 64bit ARM Emulator due to the same error
```
Failed to spawn: unable to inject library into process without libc   
```

Frida Version: 15.1.17
Frida-tools Version: frida-tools-10.6.1
ubuntu: 18.04.1
